src/fuzzers/cpp/CMakeLists.txt

```diff
@@ -49,31 +49,34 @@ else()
 endif()
 
 foreach(fuzzerName IN LISTS ALL_LOG4CXX_FUZZERS)
-  set(PROGRAM_NAME "${fuzzerName}")
-  add_executable(${PROGRAM_NAME} ${fuzzerName}.cpp)
-  target_compile_definitions(${PROGRAM_NAME}
-    PRIVATE
-      ${FUZZER_COMPILE_DEFINITIONS}
-      ${LOG4CXX_COMPILE_DEFINITIONS}
-      ${APR_COMPILE_DEFINITIONS}
-      ${APR_UTIL_COMPILE_DEFINITIONS})
-  target_include_directories(${PROGRAM_NAME}
-    PRIVATE
-      ${CMAKE_CURRENT_LIST_DIR}
-      $<TARGET_PROPERTY:log4cxx,INCLUDE_DIRECTORIES>)
-  target_link_libraries(${PROGRAM_NAME}
-    PRIVATE
-      ${FUZZER_SANITIZE_FLAGS}
-      log4cxx
-      ${APR_UTIL_LIBRARIES}
-      ${EXPAT_LIBRARIES}
-      ${APR_LIBRARIES}
-      ${APR_SYSTEM_LIBS})
-  if(WIN32)
-      set_target_properties(${PROGRAM_NAME}
-        PROPERTIES
-          VS_DEBUGGER_ENVIRONMENT "PATH=${ESCAPED_PATH}"
-          VS_DEBUGGER_WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
-          FOLDER Fuzzers)
-  endif()
+  foreach(logchar IN LISTS FUZZ_CHARS)
```

Comment:
LOG4CXX_CHAR must be set at the build directory level. Its value is [written into log4cxx.h](https://github.com/apache/logging-log4cxx/blob/b0775e7d4208d39007c33f7bc04f61eacff7979e/src/main/include/log4cxx/log4cxx.h.in#L48)

Reply:
Thanks. Have updated the PR.